docs/architecture/adr-057-RPC.md

```diff
@@ -0,0 +1,97 @@
+# ADR 057: RPC
```

Comment:
Can we call this ADR "RPC Encoding" to differentiate it from other conversations we might have about encoding?

Reply:
I'm wondering if we should have two ADR's? One for which RPC technology to use and the second one for which encoding to use if JSON-RPC is picked. 

I do agree RPC is very vague. Maybe RPC technology or something along those lines?